boards/arm/fvp_baser_aemv8r_aarch32/fvp_baser_aemv8r_aarch32_defconfig

```diff
@@ -25,3 +25,4 @@ CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
 CONFIG_CACHE_MANAGEMENT=y
+CONFIG_DCACHE=n
```

Comment:
Why not setting `CONFIG_CACHE_MANAGEMENT=n` instead?

Reply:
Because I want to keep the I-cache to be enabled.

Reply:
alright, makes sense